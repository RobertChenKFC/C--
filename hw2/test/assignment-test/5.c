The token | by itself should be illegal
