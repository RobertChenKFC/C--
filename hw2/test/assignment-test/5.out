[Error] Lexer: unrecognized token | on line 1
